Microsoft.Azure.CosmosRepository/test/Providers/DefaultCosmosContainerProviderTests.cs

```diff
@@ -16,94 +20,107 @@ namespace Microsoft.Azure.CosmosRepositoryTests.Providers
 {
     public class DefaultCosmosContainerProviderTests
     {
-        readonly ILoggerFactory _loggerFactory = new LoggerFactory();
         readonly Mock<IOptions<RepositoryOptions>> _options = new ();
-        private readonly Mock<ICosmosItemConfigurationProvider> _itemConfigurationProvider = new();
+        readonly Mock<ICosmosItemConfigurationProvider> _itemConfigurationProvider = new();
+        readonly Mock<IRepositoryOptionsValidator> _repositoryOptionsValidator = new();
+        readonly Mock<CosmosClient> _cosmosClient = new();
+        private readonly Mock<Database> _database = new();
+        private readonly Mock<Container> _container = new();
+        private readonly Mock<DatabaseResponse> _databaseResponse = new();
+        private readonly Mock<ContainerResponse> _containerResponse = new();
+        private readonly RepositoryOptions _repositoryOptions = new();

```

Comment:
```suggestion
        readonly Mock<Database> _database = new();
        readonly Mock<Container> _container = new();
        readonly Mock<DatabaseResponse> _databaseResponse = new();
        readonly Mock<ContainerResponse> _containerResponse = new();
        readonly RepositoryOptions _repositoryOptions = new();
```

Inconsistent access modifier defs, I prefer to omit defaults.